### ISSUE TITLE
lb_big_int_to_llvm bug

### ISSUE THREAD

**Issue by SentientCoffee** [odin-lang/Odin#1074]
## Context

Trying to build [this project](https://github.com/SentientCoffee/CappuccinoEngine-v3) after pulling and building `odin version dev-2021-08:7afc3672`, there is an assertion in `lb_big_int_to_llvm`

* Operating System: Windows 10 x86-64
* Please paste `odin version` output: `dev-2021-08:7afc3672`

## Expected Behavior

The program should build normally. This was the case when using odin version dev-2021-07

## Current Behavior

The program asserts and does not build.

## Failure Information (for bugs)

When debugging the odin exe, it seemed to have something to do with bitsets. Didn't look to much into it but might be a good place to start looking maybe.

### Steps to Reproduce

1. Pull latest commit from https://github.com/SentientCoffee/CappuccinoEngine-v3
2. Run `capp_build.bat`
3. See assertion failure

### Failure Logs

```
C:\dev\odin\src\llvm_backend_const.cpp(299): Assertion Failure: `sz >= max_count` max_count: 5, sz: 4, written: 0
```

### Screenshots
Assertion location:
![image](https://user-images.githubusercontent.com/22991229/129083517-d6f7fd99-7ae4-4038-9b97-1f33dc8c2b20.png)
Call stack:
![image](https://user-images.githubusercontent.com/22991229/129083555-2e65df28-714a-4bd8-b8d0-cb5a8f3186b0.png)

**Comment by SentientCoffee** [comment]
Looking a little more into it, it seems this bug starts happening from `odin version dev-2021-08:74c019f2`. I can compile my code just fine with `odin version dev-2021-08:698eeaf7`.

**Comment by gingerBill** [comment]
Fixed in 0e84e067562c490387cd221eccf3e26e14d09df5.
BUT this will reveal a bug in your code.
